devices/Ft6xx6x/Rotation.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Ft6xx6x
+{
+    /// <summary>
+    /// The touch coordinates rotation relative to the default orientation.
+    /// </summary>
+    public enum Rotation
```

Comment:
rather than this file, can you add the graphics core nuget and use this: https://github.com/nanoframework/nanoFramework.Graphics/blob/main/nanoFramework.Graphics.Core/nanoFramework/UI/DisplayOrientation.cs